libs/gltfio/materials/ubershader.mat.in

```diff
@@ -3,6 +3,9 @@ material {
     requires : [ uv0, uv1, color ],
     shadingModel : ${SHADINGMODEL},
     blending : ${BLENDING},
+    depthWrite : true,
+    doubleSided : true,
```

Comment:
I can why you want to set doubleSided at runtime :)

Reply:
Interestingly, LittlestTokyo requires us to honor doubleSided properly, otherwise the model does not look good.